force-app/main/default/classes/DMLOperations.cls

```diff
@@ -145,63 +211,154 @@ public with sharing class DMLOperations {
      * @param oppNames The list of Opportunity names.
      */
     public static void upsertOpportunities(String accountName, List<String> oppNames) {
-        // Loop through each Opportunity name
-            // Create an Opportunity with the given name
+    // Look for account "accountName"; upsert if not there
+    Account account;
```

Comment:
When possible, try to use abbreviations for sObjects (eg, `acc`). It can be confusing to read code when there are variable names that mimic the class names (eg, Account vs account).